openff/toolkit/tests/test_toolkits.py

```diff
@@ -1657,8 +1657,10 @@ def test_assign_fractional_bond_orders_invalid_method(self):
             )
 
     @pytest.mark.slow
-    def test_substructure_search_on_large_molecule(self):
-        """Test OpenEyeToolkitWrapper substructure search when a large number hits are found"""
+    def test_max_substructure_matches_can_handle_large_molecule(self):
+        """Test OpenEyeToolkitWrapper substructure search handles more than the default of MaxMatches = 1024
+        See https://github.com/openforcefield/openff-toolkit/pull/509
```

Comment:
(not blocking) Let's make this consistent with the other docstring
```suggestion
        See https://github.com/openforcefield/openff-toolkit/pull/509 .
```